.github/workflows/release.yml

```diff
@@ -0,0 +1,50 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - "v*"
+
+permissions:
+  contents: write
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      -
+        name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+
+      - name: Install cross-compile package
+        run: |
+          sudo apt update
```

Comment:
It's good practice to update before dependencies installation, but I'm not sure if we really need it